src/frontend/src/session.rs

```diff
@@ -1367,6 +1399,24 @@ impl SessionImpl {
         self.temporary_source_manager.lock().clone()
     }
 
+    pub fn create_staging_table(&self, table: TableCatalog) {
+        self.staging_catalog_manager
+            .lock()
+            .create_table(table.name.clone(), table);
+    }
+
+    pub fn get_staging_table(&self, name: &str) -> Option<TableCatalog> {
```

Comment:
Where do we call this method? It seems we only create the staging table and drop it.

Reply:
Oh, I cloned the staging catalog manager and used `get_table` directly during binding stage. Let me remove it for now.